app/src/main/java/org/groundplatform/android/repository/SurveyRepository.kt

```diff
@@ -76,6 +77,7 @@ constructor(
 
     // Wait for survey to be updated. Else throw an error after timeout.
     withTimeout(ACTIVATE_SURVEY_TIMEOUT_MILLS) {
+      Timber.e("Waiting for survey to get updated")
```

Comment:
Why are we logging this as an error? Isn't that incorrect?

Reply:
As for the error in issue description, from here we get to know is this is causing the error or not.

Also, this way we get to know if any specific survey getting too much time to load or anything else.